javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/DescriptorTest.java

```diff
@@ -17,18 +22,35 @@
 class DescriptorTest extends AbstractResolutionTest {
 
     @Test
-    void descriptorTest() {
-        String code = 
+    void descriptorTest() throws IOException {
+        String code =
+                "import javassist.ClassPool;\n" +
                 "public class A {\n" +
                 "  A(int i, double d, Thread t) {}\n" +
+                "  public enum TestEnum {\n" +
+                "    TEST_ENUM(\"test\");" +
+                "    private String a;\n" +
+                "    private TestEnum(String a) {\n" +
+                "      this.a = a;\n" +
+                "    }\n" +
+                "  }\n" +
                 "  Object m(int i, double d, Thread t) {return new Object();}\n" +
                 "  void m(int i, double d, Thread t) {}\n" +
                 "  int[] m(int i, double d, Thread t) {return new int[] {};}\n" +
                 "  long[][] m(int i, double d, Thread t) {return new long[][] {};}\n" +
-                "  void m() {}\n" +
+                "  void m() {\n" +
+                "    System.out.println(\"a\");\n" +
+                "    ClassPool cp = ClassPool.getDefault();\n" +
+                "    TestEnum.valueOf(\"TEST_ENUM\");\n" +
+                "    TestEnum.values();\n" +
+                "  }\n" +
                 "}";
+
+        Path javassistJar = adaptPath("src/test/resources/issue3808/javassist-3.29.2-GA.jar");
```

Comment:
Do not include the javassists library in the project resources

Reply:
I have removed the jar and used the javassistmethoddecl.jar from the resources.